core/src/main/java/org/bitcoinj/store/WalletProtobufSerializer.java

```diff
@@ -76,6 +76,10 @@
     private static final Logger log = LoggerFactory.getLogger(WalletProtobufSerializer.class);
     /** Current version used for serializing wallets. A version higher than this is considered from the future. */
     public static final int CURRENT_WALLET_VERSION = Protos.Wallet.getDefaultInstance().getVersion();
+
+    // 512 MB
+    private static final Integer walletSizeLimit = 512 * 1024 * 1024;
```

Comment:
Can you make this an int (native type) and change the name to allcaps "WALLET_SIZE_LIMIT"?